### DISCUSSION TITLE
Advanced music settings

### DISCUSSION THREAD

**Discussion by waffleeiron** [BedrockTweaks/Files#338] [Ideas]
A tweak that allows one to enable/disable music to play in different biomes, dimensions or gives you presets of music settings for example to let only c418 music play, only lena raine music, only music pre update xyz, there's a lot of possibilities

**Comment by SquatchHunter** [comment]
not sure if possible, plus massively out of scope for this project when addons like Music+ already exist

**Comment by IdotIcom** [comment]
https://youtu.be/ozaXz-hjbDE?si=hCZZZ652SZvTNttn
